packages/block-library/src/latest-comments/edit.js

```diff
@@ -25,83 +24,59 @@ const MIN_COMMENTS = 1;
  */
 const MAX_COMMENTS = 100;
 
-class LatestComments extends Component {
-	constructor() {
-		super( ...arguments );
+export default function LatestComments( { attributes, setAttributes } ) {
+	const {
+		commentsToShow,
+		displayAvatar,
+		displayDate,
+		displayExcerpt,
+	} = attributes;
```

Comment:
You can destructure `attributes` in the above destructuring.

```js
{ attributes, attributes: { commentsToshow, ....... } setAttributes }
```

Reply:
Actually, it would be `{ attributes: { commentsToShow, (etc) }, setAttributes }`. No need to duplicate `attributes`; actually, I think that would cause a linting error.